### ISSUE TITLE
[Bug] ViewModel property value gets set 2 - 3 times in a row

### ISSUE THREAD

**Issue by ctrlemo** [PrismLibrary/Prism#1905]
### Steps to Reproduce

1. Create a ViewModel and hook it to a View with Prism.Ioc.IContainerRegistry
2. ViewModel should implement all of these BindableBase, IInitialize, INavigatedAware, IDestructible, and also call BindableBase.SetProperty in each Property in ViewModel
3. Try to navigate to the View, and in the ViewModel's Initialize function set the properties with values from the INavigationParameters parameters being passed in.
4. Place breakpoint on the properties set function in the ViewModel. You will notice each property gets set 2 or sometimes even 3 times. And the 3rd time the value being passed in is null.

### Expected Behavior
ViewModel's Properties get set 1 time.
### Actual Behavior
ViewModel's Properties get set 2-3 times with same value or sometimes null
### Basic Information

- Version with issue: Prism.DryIoc.Forms 7.2.0.1367
- Last known good version: Prism.DryIoc.Forms 7.1.0.431
- Xamarin.Forms version: 4.2.0.778463

- IDE: Visual Studio for Mac

**Comment by umitnuri** [comment]
This needs reproduction repo. Looks like your code issue

**Comment by dansiegel** [comment]
This seems like either a problem with Visual Studio not hitting your debug point correctly or something in your code. Either way you provided no reproduction.

**Comment by ctrlemo** [comment]
Hi @dansiegel 
You were right. it's been a while since I worked with C#, Xamarin and bindings between View and ViewModel.

Turns out the double calls to set properties is normal behaviour when using TwoWay bindings.
The 3 calls to set a property is happening because of a Picker I am using to that of which both it's SelectedItem and SelectedIndex are hooked up with TwoWay binding and this was causing the 3rd call to set with selectedItem being Null upon loading the View and VIewModel.

So I apologize for opening up the bug.
